ocaml/xenopsd/test/test_topology.ml

```diff
@@ -28,7 +32,20 @@ let make_numa_amd ~cores_per_numa =
   let cpu_to_node =
     Array.init (cores_per_numa * numa) (fun core -> core / cores_per_numa)
   in
-  (cores_per_numa * numa, NUMA.make ~distances ~cpu_to_node)
+  match NUMA.make ~distances ~cpu_to_node with
+  | None ->
+      Alcotest.fail "Synthetic matrix can't fail to load"
+  | Some d ->
+      (cores_per_numa * numa, d)
+
+let make_numa_unreachable ~cores_per_numa =
+  let numa = 2 in
+  (* 4294967295 is exactly (2ˆ32) - 1, meaning the node is unreachable *)
+  let distances = [|[|10; 4294967295|]; [|4294967295; 4294967295|]|] in
```

Comment:
This is fine, but you could also write `1 lsl 32 - 1` or `0xffff_ffff`

Reply:
This is changed in a following commit